CIP-????/README.md

```diff
@@ -0,0 +1,97 @@
+---
+CPS: ????
+Title: Lack of builtin data structures with good asymptotic performance in Plutus Core
```

Comment:
```suggestion
Better builtin datastructures in Plutus
```
The current PR title sounds both broad and specific enough to cover this: if not this one, we would need a final title of around this length.  Your stating the "problem" in terms of what we are hoping to achieve ("Better XXX") I think works well here.

Reply:
I'm not sure. I think the problem _is_ broader, I think it's jumping to conclusions a bit to structure it directly around the solution we're currently thinking of!

Reply:
OK sounds like we're good to go with the broad, brief title (our CIP titles read more like newspaper headlines than academic papers)